pkgs/misc/emulators/dolphin-emu/master.nix

```diff
@@ -18,29 +18,29 @@ let
     categories = "Game;Emulator;";
     startupNotify = "false";
   };
-in stdenv.mkDerivation rec {
+in mkDerivation rec {
   name = "dolphin-emu-${version}";
   version = "5.0-10751";
```

Comment:
Please make sure to change the version when bumping the revision! The latest version number can be found on dolphin's website in the nightly builds. For the commit you specified, that version would be 5.0-10879

Reply:
good catch, thanks